### ISSUE TITLE
Get just phone number without country code

### ISSUE THREAD

**Issue by RShah512** [thegamenicorus/react-native-phone-input#79]
Is there a way to separate out country code from the phone number?

**Comment by yasir-netlinks** [comment]
@RShah512 hi , have you found a way to do this ?

**Comment by boppuh** [comment]
Use string replace to replace the + and the first occurrence of the country code with an empty string. 
Auto-format must be false, or you must replace the other formatting characters and blank spaces with an empty string, as well.

```
const countryCode = this.phone.getCountryCode()
let phoneNumber = this.phone.getValue()
phoneNumber = phoneNumber.replace('+','')
phoneNumber = phoneNumber.replace(countryCode, '')
```

**Comment by RShah512** [comment]
@yasir-netlinks Yes, I used getCountryCode function onSelectCountry property and then like @boppuh answer, I used replace to remove the '+' sign.
